site/content/docs/commands/pipeline-deploy.en.md

```diff
@@ -0,0 +1,19 @@
+# pipeline deploy (same as pipeline update)
```

Comment:
We should probably try to reuse the template so that we don't need to update two places when we add new flags.

Reply:
Okay! I combined the two pages...let me know if it's too ugly/weird/bad.

Reply:
Got rid of `pipeline-update` pages and added redirects so if users try to go to the old ones, they'll get redirected to the new `pipeline-deploy` pages.